manifests/release/conformance.yaml.in

```diff
@@ -0,0 +1,13 @@
+sonobuoy-config:
+  driver: Job
+  plugin-name: kubevirt-conformance
+  result-format: junit
+spec:
+  command:
+  - /usr/bin/conformance
+  image: {{.DockerPrefix}}/conformance:{{.DockerTag}}
+  name: plugin
+  resources: {}
+  volumeMounts:
+  - mountPath: /tmp/results
+    name: results
```

Comment:
running 
```
sonobuoy gen plugin --name kubevirt-conformance --cmd /usr/bin/conformance --image IMAGE -f junit > manifests/release/conformance.yaml.in
sed -i 's#IMAGE#{{.DockerPrefix}}/conformance:{{.DockerTag}}#' manifests/release/conformance.yaml.in
```

gives another `\n` to this file from some reason. could this be an issue (prorclain/check wise)?

Reply:
It is not autogenerated, so I don't think it would cause any harm. I can keep the line if you think it would be better, I don't have a strong feelings about it.

Reply:
if you don't think it could cause an issue then I prefer what you did. 
we can always change it later should a problem arises.